test/python/unit/api/test_solver.py

```diff
@@ -944,42 +944,23 @@ def test_declare_sort(solver):
 
 def test_define_fun(solver):
     bvSort = solver.mkBitVectorSort(32)
-    funSort1 = solver.mkFunctionSort([bvSort, bvSort], bvSort)
-    funSort2 = solver.mkFunctionSort(solver.mkUninterpretedSort("u"),\
-                                     solver.getIntegerSort())
+    funSort = solver.mkFunctionSort(solver.mkUninterpretedSort("u"),
+                                    solver.getIntegerSort())
     b1 = solver.mkVar(bvSort, "b1")
-    b11 = solver.mkVar(bvSort, "b1")
     b2 = solver.mkVar(solver.getIntegerSort(), "b2")
-    b3 = solver.mkVar(funSort2, "b3")
+    b3 = solver.mkVar(funSort, "b3")
     v1 = solver.mkConst(bvSort, "v1")
-    v2 = solver.mkConst(solver.getIntegerSort(), "v2")
-    v3 = solver.mkConst(funSort2, "v3")
-    f1 = solver.mkConst(funSort1, "f1")
-    f2 = solver.mkConst(funSort2, "f2")
-    f3 = solver.mkConst(bvSort, "f3")
+    v2 = solver.mkConst(funSort, "v2")
     solver.defineFun("f", [], bvSort, v1)
     solver.defineFun("ff", [b1, b2], bvSort, v1)
-    solver.defineFun(f1, [b1, b11], v1)
     with pytest.raises(RuntimeError):
         solver.defineFun("ff", [v1, b2], bvSort, v1)
     with pytest.raises(RuntimeError):
-        solver.defineFun("fff", [b1], bvSort, v3)
+        solver.defineFun("fff", [b1], bvSort, v2)
     with pytest.raises(RuntimeError):
-        solver.defineFun("ffff", [b1], funSort2, v3)
+        solver.defineFun("ffff", [b1], funSort, v2)
     # b3 has function sort, which is allowed as an argument
     solver.defineFun("fffff", [b1, b3], bvSort, v1)
-    with pytest.raises(RuntimeError):
```

Comment:
I tried. But most of those test cases are not applicable to the other method because they check that the sort of the function term matches with the sorts of the parameter list and the body. The remaining ones are already tested.

Reply:
Ok, sounds good.